BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudClient.cs

```diff
@@ -281,6 +281,7 @@ public bool Initialized
         public void EnableCompression(bool compress)
```

Comment:
So in Unity, it's always accepted compressed responses? I think this needs to be a separate call.
`EnableCompressedResponses()`. The idea is, platform can control if they want compressed responses or not, independently from sending compressed.

Reply:
By default the compressed variable values are false until the client calls EnableCompressedResponses() so in unity it won't always accept compressed responses.